gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
```

Comment:
@sshropshire This bumps us to require Flamingo | 2022.2.1, correct? (Looking [here](https://developer.android.com/studio/releases#android_gradle_plugin_and_android_studio_compatibility))

If so - we should call out the required Android Studio version in the README

Reply:
That's a good call. I'll update to make sure, I could see it breaking something.